cluster-autoscaler/cloudprovider/openshiftmachineapi/machineapi_utils.go

```diff
@@ -143,3 +151,42 @@ func machineSetIsOwnedByMachineDeployment(machineSet *MachineSet, machineDeploym
 	}
 	return false
 }
+
+func scaleFromZeroEnabled(annotations map[string]string) bool {
+	cpu := annotations[cpuKey]
+	mem := annotations[memoryKey]
+	gpu := annotations[gpuKey]
+
+	if cpu != "" && mem != "" && gpu != "" {
```

Comment:
Suggestion to only require CPU and Memory since GPU can have a sane default of `0` and we don't currently support GPUs across all providers (eg GCP)
```suggestion
	if cpu != "" && mem != ""  {
```